examples/getting_started/pendulum.py

```diff
@@ -19,7 +21,61 @@
     InitialGuess,
     ObjectiveFcn,
     Objective,
+    PlotType,
+    ObjectiveList,
 )
+def plot_objectives(ocp):
```

Comment:
I am not sure I understand the purpose of preparing everything in gui.py but actually leaves the user all the work to do. Is this an involuntary commit in pendulum.py (I mean that you are working on user side because it is easier(?) and plan to move this plot_objectives stuff in another file later on)?

Reply:
A thought you told me to do it this way... I will move it when it is ready to merge.

Reply:
Done!